src/webcomponents/variant/interpretation/variant-interpreter-review-primary.js

```diff
@@ -357,101 +357,109 @@ export default class VariantInterpreterReviewPrimary extends LitElement {
+                        `,
                     },
                     {
                         id: "cohortStats",
                         name: "Cohort Stats",
-                        render: (variant, active, opencgaSession) => {
-                            return html`
+                        render: (variant, active, opencgaSession) => html`
                             <variant-cohort-stats
-                                    .opencgaSession="${opencgaSession}"
-                                    .variantId="${variant.id}"
-                                    .active="${active}">
-                            </variant-cohort-stats>`;
-                        }
+                                .opencgaSession="${opencgaSession}"
+                                .variantId="${variant.id}"
+                                .active="${active}">
+                            </variant-cohort-stats>
+                        `,
                     },
                     {
                         id: "samples",
                         name: "Samples",
                         render: (variant, active, opencgaSession) => html`
-                        <variant-samples
-                            .opencgaSession="${opencgaSession}"
-                            .variantId="${variant.id}"
-                            .active="${active}">
-                        </variant-samples>`,
+                            <variant-samples
+                                .opencgaSession="${opencgaSession}"
+                                .variantId="${variant.id}"
+                                .active="${active}">
+                            </variant-samples>
+                        `,
                     },
                     {
                         id: "beacon",
                         name: "Beacon",
-                        render: (variant, active, opencgaSession) => {
-                            return html`
+                        render: (variant, active, opencgaSession) => html`
                             <variant-beacon-network
-                                    .variant="${variant.id}"
-                                    .assembly="${opencgaSession.project.organism.assembly}"
-                                    .config="${this.beaconConfig}"
-                                    .active="${active}">
-                            </variant-beacon-network>`;
-                        }
+                                .variant="${variant.id}"
+                                .assembly="${opencgaSession.project.organism.assembly}"
+                                .config="${this.beaconConfig}"
+                                .active="${active}">
+                            </variant-beacon-network>
+                        `,
+                    },
+                    {
+                        id: "exomiser",
+                        name: "Exomiser",
+                        render: (variant, active) => html`
```

Comment:
To double-check if the visible function is needed here:
```suggestion
    visible: () => {
        return this.clinicalAnalysis?.interpretation?.method?.name === "interpretation-exomiser";
    },
```